src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -444,6 +433,22 @@ void TranslateSession::translate_graph(const ov::frontend::InputModel::Ptr& inpu
         }
     }
 
+    if (saved_model_inputs.get() && saved_model_inputs->size() > 0) {
+        for (auto param : params) {
+            if (!apply_saved_model_names(param, saved_model_inputs)) {
```

Comment:
please add a comment that unused placeholders are not mapped to user's input names